docs/source/tracking.rst

```diff
@@ -57,7 +57,8 @@ API.
 Where Runs Are Recorded
 =======================
 
-MLflow runs are recorded either locally in files or remotely to a tracking server.
+MLflow runs are recorded either locally in files, or SQLAlchemy compatible database, or remotely
```

Comment:
How do you feel about using a bulleted list? The phrasing here is a little hard to parse.

```
Users can record MLflow runs in the following locations:

- **Locally** as files
- **Locally** as entries in a SQLAlchemy-compatible database
- **Remotely** to a tracking server
```